README.md

```diff
@@ -32,6 +33,24 @@ API provides the following URLs for working with remote storage configurations:
 3. User with name `{{username}}`, password `{{password}}`, remote-storage.all permissions should be created on FOLIO. After that apikey can
    be generated for making calls to Edge CaiaSoft API.
 
+#### Rancher and kubernetes deployment
+1. Check that mod-remote-storage has been installed and has been registered to okapi.
+2. Create a new user named `stagingDirector` in FOLIO. You may also use `diku_admin` for testing and avoid this step.
+3. Create a secret in the rancher cluster. Make the key of this secret `ephemeral.properties` and the value similar to `secureStore.type=Ephemeral tenants=diku diku=diku_admin,admin,stagingDirector`.
```

Comment:
Does this use spaces or line breaks as separators?

Reply:
The list above my changes had three spaces inserted in the middle of lines probably to try to make the list be indented. I removed those to make the file more consistent.